demo/model_app.py

```diff
@@ -231,6 +235,7 @@ class CommandId:
     ),
     types.Action(
         id="about",
+        icon=f"file://{ABOUT_ICON_PATH}",
```

Comment:
Hi there, thank you for pushing foward this! Quick comment, I think there is a need (at least on Windows) to put three slashes here (`///`) otherwise no icon is shown:

![imagen](https://github.com/user-attachments/assets/965879cc-71fa-4dd1-81ba-afe2623a73f9)

Reply:
i see, apologies for changing that!  I thought the third slash was an oversight (since there's also a preceding slash for posix systems in ABOUT_ICON_PATH).

Reply:
Oh I see, maybe we should show over the example the need to handle the slashes for `file:` depending on the platform? Checking https://en.wikipedia.org/wiki/File_URI_scheme I see that on Unix the preceding slash is taken into account for the three slashes the URI has before the actual file path :thinking:

Reply:
Based on the fact that your previous version was working fine (and tests are passing again after reverting it), it appears to be fine for posix systems to have four slashes (`file:////Users...`).  So I'm fine leaving it as is and not doing anything platform specific.  But I suppose we could add a note somewhere?  Your call, I'm fine just merging

Reply:
Woah I said "fine" a lot in there 😅

Reply:
Over the last commit [adffa9b](https://github.com/pyapp-kit/app-model/pull/219/commits/adffa9b330f5d3834616e2059c433f0777a5dd1e) I added some info about the possibility to use a `.svg` file path over the Action `icon` field with a note just in case. Seems like an approapiate place but let me know what you think!

Reply:
perfect!